README.md

```diff
@@ -58,21 +60,20 @@ We use Ruby's built-in logger to output messages to standard output. Learn more
 
 Here's an example of output from the courses scraper:
 
-```
+```log
 [2018-10-18 01:35:01] INFO  (courses_scraper): Searching for courses in term 201801
 [2018-10-18 01:35:02] INFO  (courses_scraper): 178 department/semesters so far
 [2018-10-18 01:35:02] INFO  (courses_scraper): Searching for courses in term 201805
 [2018-10-18 01:35:03] INFO  (courses_scraper): 301 department/semesters so far
 ```
 
-The formatting for outputted messages is as follows:`[DATE TIME] LOG_LEVEL (PROGRAM_NAME): {MESSAGE}`
+The formatting for outputted messages is as follows: `[DATE TIME] LOG_LEVEL (PROGRAM_NAME): {MESSAGE}`
 
-An example of a log call in ruby:
-`logger.info(prog_name) {"MESSAGE"}`
+An example of a log call in ruby: `logger.info(prog_name) {"MESSAGE"}`
 
 You should use Ruby's built-in log-levels where appropriate, when displaying errors you should use logger.error, when displaying information you should use logger.info, and so on.
 
-Our logger implementation is located at the `scraper_common.rb` file located at `$app/scraper_common.rb`
+Our logger implementation is located at the `scraper_common.rb` file located at `$app/scraper_common.rb`.
```

Comment:
it's actually `app/scrapers/scraper_common.rb`

Reply:
Didn't actually touch this, but can fix